### ISSUE TITLE
Update UI

### ISSUE THREAD

**Issue by nikhilbansal97** [nikhilbansal97/World-Now#1]
Update the UI of the App according to Material Design

**Comment by reesea** [comment]
Hi, I'm a beginner, can I work on this?

**Comment by nikhilbansal97** [comment]
Yes, sure @reesea. Fork the repository, create a new branch and start working on it!

**Comment by Kapil706** [comment]
Hi I am kapil Chaudhary and I want to work on this 
Upto which stage this application is completed and Is it necessary to work in kotlin only or i can work in java because i more comfortable with java but can do kotlin also

**Comment by nikhilbansal97** [comment]
@Kapil706 I would prefer if you work in Kotlin. And @reesea has started working on it. If @reesea does not raise a PR after 1 week, then I will assign it to you

**Comment by Kapil706** [comment]
@nikhilbansal97   I just clone application in android studio using terminal and when i opened it is showing this 

Could not get unknown property 'Guardian_ApiKey' for BuildType_Decorated{name=debug, debuggable=true, testCoverageEnabled=false, jniDebuggable=false, pseudoLocalesEnabled=false, renderscriptDebuggable=false, renderscriptOptimLevel=3, minifyEnabled=false, zipAlignEnabled=true, signingConfig=SigningConfig_Decorated{name=debug, storeFile=C:\Users\kapil\.android\debug.keystore, storePassword=android, keyAlias=AndroidDebugKey, keyPassword=android, storeType=C:\Users\kapil\.android\debug.keystore, v1SigningEnabled=true, v2SigningEnabled=true}, embedMicroApp=false, mBuildConfigFields={}, mResValues={}, mProguardFiles=[], mConsumerProguardFiles=[], mManifestPlaceholders={}} of type com.android.build.gradle.internal.dsl.BuildType.
Open File
What Should i Do ?
I checked twice on stackoverflow but do not find any solution

**Comment by nikhilbansal97** [comment]
You can just add your own Guardian API key in the build.gradle file

**Comment by reesea** [comment]
I'll withdraw. I'm not that familiar with kotlin.

**Comment by nikhilbansal97** [comment]
@reesea Okay, cool. No issue. @Kapil706 You can work on this.

**Comment by Kapil706** [comment]
No problem but what about guardian API Key

**Comment by nikhilbansal97** [comment]
> No problem but what about guardian API Key

@Kapil706 You can create a Guardian API key [here](https://open-platform.theguardian.com/access/) and replace it in the `build.gradle` file like this:

<img width="1280" alt="screen shot 2018-10-02 at 8 54 28 pm" src="https://user-images.githubusercontent.com/18311406/46358785-6313eb80-c685-11e8-8542-1b8353882948.png">

**Comment by NalediMadlopha** [comment]
I have created an issue to add a section in the README.md to make it easy for people like @Kapil706 to get up and running with the project.

**Comment by nikhilbansal97** [comment]
> I have created an issue to add a section in the README.md to make it easy for people like @Kapil706 to get up and running with the project.

@NalediMadlopha Thanks Man! 😄
